packages/editor/src/editor-ui/editor-tooltip.tsx

```diff
@@ -19,14 +21,22 @@ export function EditorTooltip({
     isMac ? '⌘' : strings.keys.ctrl
   )
 
+  const positionClasses = {
```

Comment:
This could have been done before using the `className` property. Having a `position` property is fine with me. Did you check for previous usages of EditorTooltip, to see if there were any places where you could replace the `className` method with your new method of positioning?